dataikuapi/dss/project.py

```diff
@@ -79,17 +79,25 @@ def move_to_folder(self, folder):
     # Project deletion
     ########################################################
 
-    def delete(self, drop_data=False):
+    def delete(self, clear_managed_datasets=False, clear_output_managed_folders=False, clear_job_and_scenario_logs=True, **kwargs):
```

Comment:
I found many instances in integration tests (and beyond) where we call this with `drop_data=True` or just with one positional argument `True`. Should I go ahead and change these calls?

Reply:
Done.